rocks-db/src/columns/offchain_data.rs

```diff
@@ -0,0 +1,141 @@
+use crate::column::TypedColumn;
+use crate::generated::offchain_data_generated::off_chain_data as fb;
+use crate::key_encoders::{decode_string, encode_string};
+use crate::{Result, ToFlatbuffersConverter};
+use entities::models::OffChainDataGrpc;
+use flatbuffers::{FlatBufferBuilder, WIPOffset};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub enum StorageMutability {
+    #[default]
+    Immutable,
+    Mutable,
+}
+
+impl StorageMutability {
+    pub fn is_mutable(&self) -> bool {
+        match self {
+            StorageMutability::Immutable => false,
+            StorageMutability::Mutable => true,
+        }
+    }
+}
+
+impl From<&str> for StorageMutability {
+    fn from(storage_mutability: &str) -> Self {
+        if storage_mutability.is_empty()
+            || storage_mutability.starts_with("")
```

Comment:
This will match any string imaginable, right?

Reply:
Right ⚰️